General/ProgressBar.md

```diff
@@ -70,4 +68,4 @@ UIManager default
 	]  ]
```

Comment:
Don't we need to declare `lastUpdate` var in the block?

Reply:
It's done.
`during: [ :bar | | lastUpdate | `